packages/xo-server/src/api/metadata-backup.js

```diff
@@ -101,3 +101,48 @@ runJob.params = {
     type: 'string',
   },
 }
+
+export async function listMetadataBackups({ remotes }) {
```

Comment:
No need to add `MetadataBackup`, this method is already defined in the `metadataBackup` API namespace.